Refactor cukinia configuration and update documentation (#11)
seapath/cukinia-tests: README.md

# cukinia-tests

This repository is a part of the [SEAPATH](https://www.lfenergy.org/projects/seapath/) project.

> It contained Cukinia's tests, which can be run on SEAPATH machines.

Reviewer: It contains

Reviewer: ```suggestion
It contains Cukinia's tests, which can be run on SEAPATH machines.
```

> These tests can be run and deployed using SEAPATH ansible playbook:

Reviewer: The SEAPATH ansible

Reviewer: ```suggestion
These tests can be run and deployed using the SEAPATH ansible playbook:
```

* [test_deploy_cukinia.yaml](https://github.com/seapath/ansible/blob/debian-main/playbooks/test_deploy_cukinia.yaml "test_deploy_cukinia.yaml") to install Cukinia on a machine
* [test_deploy_cukinia_tests.yaml](https://github.com/seapath/ansible/blob/debian-main/playbooks/test_deploy_cukinia_tests.yaml "test_deploy_cukinia_tests.yaml") to deploy tests
* [test_run_cukinia.yaml](https://github.com/seapath/ansible/blob/debian-main/playbooks/test_run_cukinia.yaml "test_run_cukinia.yaml") to run the tests

> Test are installed in /etc/cukinia. It can be run manually with the cukinia command:

Reviewer: ```suggestion
Tests are installed in /etc/cukinia. They can be run manually with the cukinia command:
```
